### PR TITLE
Fix typos, broken links, hints not showing up etc.

### PR DESCRIPTION
**Pull request by angelikatyborska**
I just went through the first 4 concept exercises in Ruby on exercism.lol for the first time. I noticed a few tiny details that need improvement. I'm leaving my feedback in a form of a PR that already fixes those things. The commit history should describe well the changes I'm proposing. I'll revert anything you don't really want 🙂 in general:

- some random typos
- some hints not showing up because there were paragraphs between list items or because the task name was different than in instructions

### CODE REVIEW

File: exercises/concept/amusement-park/.docs/hints.md [hunk 1]
```diff
@@ -21,18 +16,18 @@ About instance variables:
...
-## 3. What is the ride pass's id
+## 3. What is the ride pass id
 
 - Create a getter for the pass id.
 - It is okay and expected to return `nil` before a pass is issued to the `Attendee`.
 
 ## 4. Allow people to buy a pass
 
-- Using the `Attendee#issue_pass!` method, set the instance's state to the argument.
+- In the `Attendee#issue_pass!` method, set the instance's state to the argument.
```
Comment (angelikatyborska) [inline exercises/concept/amusement-park/.docs/hints.md:26]: "Using ..." sounded to me like I'm supposed to call that method somehow.

File: exercises/concept/boutique-inventory/.meta/exemplar.rb [hunk 2]
```diff
@@ -2,18 +2,18 @@ class BoutiqueInventory
...
   def cheap
     items.select { |item| item[:price] < 30 }
   end
- 
+
   def out_of_stock
-    items.select do |item| 
-      item[:quantity_by_size].none {|size, quantity| quanity > 0 }
+    items.select do |item|
+      item[:quantity_by_size].none {|size, quantity| quantity > 0 }
```
Comment (angelikatyborska) [inline exercises/concept/boutique-inventory/.meta/exemplar.rb:16]: This intrigues me, shouldn't this have been caught by a CI check that runs tests on all exemplar solutions?

File: exercises/concept/lasagna/.docs/hints.md [hunk 3]
```diff
@@ -26,7 +26,7 @@
 - You can use the [mathematical operator for addition][operators] to add values.
 
 [methods]: https://launchschool.com/books/ruby/read/methods
-[return]: https://www.freecodecamp.org/news/idiomatic-ruby-writing-beautiful-code-6845c830c664/
+[return]: https://www.freecodecamp.org/news/idiomatic-ruby-writing-beautiful-code-6845c830c664/#implicit-return
```
Comment (angelikatyborska) [inline exercises/concept/lasagna/.docs/hints.md:29]: This is a long article and the relevant part is far away from the beginning, so I thought it's better to link right to it.

File: exercises/concept/log-line-parser/.docs/introduction.md [hunk 4]
```diff
@@ -54,8 +54,8 @@ str[1, 3]         #=> "ome"
 str.slice(1, 3)   #=> "ome"
 ```
 
-The semantic values of strings can also be maniplulated.
-It's worth exploring the Ruby docs to learn about all the methods, but here are a few commonly used examples:
+The semantic values of strings can also be manipulated.
+It's worth exploring the [Ruby docs][docs-string] to learn about all the methods, but here are a few commonly used examples:
```
Comment (angelikatyborska) [inline exercises/concept/log-line-parser/.docs/introduction.md:58]: I could have used a link here. It's not entirely clear at this point how to find the Ruby docs. There's a link in the hints, I feel like this is too fundamental to the concept to "hide" in the hints 🤔

File: exercises/concept/boutique-inventory/.meta/exemplar.rb (hunk 2 above)
Comment (SleeplessByte) [reply exercises/concept/boutique-inventory/.meta/exemplar.rb:16]: There's even more mistakes. The function is called `none?`, not `none`. That means that `out_of_stock` is likely never hit.

Comment (SleeplessByte) [reply exercises/concept/boutique-inventory/.meta/exemplar.rb:16]:
```suggestion
      item[:quantity_by_size].none? {|size, quantity| quantity > 0 }
```

**Review by SleeplessByte — CHANGES_REQUESTED** [review]
See inline

File: exercises/concept/boutique-inventory/.meta/exemplar.rb (hunk 2 above)
Comment (angelikatyborska) [reply exercises/concept/boutique-inventory/.meta/exemplar.rb]: Got it. After that change, the tests pass, I checked manually. But it looks like the CI check is for practice exercises only.

File: exercises/concept/log-line-parser/.docs/introduction.md (hunk 4 above)
Comment (iHiD) [reply exercises/concept/log-line-parser/.docs/introduction.md:58]: Agreed.

**Review by iHiD — APPROVED** [review]
Thanks!

**Comment by iHiD** [comment]
We good to go on this, @SleeplessByte?

**Comment by SleeplessByte** [comment]
I think so haha

**Pull request merged**